Update 671fa47e415d88263d349a10.md

mayur-driod opened the pull request:
Checklist:

<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [ x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [ x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x ] My pull request targets the `main` branch of freeCodeCamp.
- [ x] I have tested these changes either locally on my machine, or GitPod.

<!--If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.-->

Closes #XXXXX

<!-- Feel free to add any additional description of changes below this line -->

made changes from "Your `script` element should be have an `src` attribute set to `script.js`." to "Your `script` element should have an `src` attribute set to `script.js`."

mayur-driod committed:
Update 671fa47e415d88263d349a10.md

mayur-driod commented:
Hi since the message above doesn't accurately address the required format I am re-writing it. 

[ x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org/).
[ x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
[x ] My pull request targets the main branch of freeCodeCamp.
[ x] I have tested these changes locally on my machine, or GitPod.
Closes #58186 

I changed from "Your script element should have an src attribute set to script.js" to "Your script element should have an src attribute set to script.js."

fix: changed the typo
feat: changed typo
docs:[671fa47e415d88263d349a10.md](https://github.com/freeCodeCamp/freeCodeCamp/pull/58186#top)

ilenia-magoni commented:
@mayur-driod you still need to address the issue of the PR not following guidelines. Also if you want to update the top post please edit it

mayur-driod commented:
#58142

Hi since the message above doesn't accurately address the required format I am re-writing it.

[ x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org/).
[ x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
[x ] My pull request targets the main branch of freeCodeCamp.
[ x] I have tested these changes locally on my machine, or GitPod.

Closes https://github.com/freeCodeCamp/freeCodeCamp/issues/58142

I changed from "Your script element should have an src attribute set to script.js" to "Your script element should have an src attribute set to script.js."

fix: changed the typo
feat: changed typo
docs:https://github.com/freeCodeCamp/freeCodeCamp/pull/58186#top

ilenia-magoni commented:
@mayur-driod the issue is still the title of the PR, but it's a bit late, this PR was closed a few weeks ago. If you want to contribute you should look into a different issue to contribute to
